src/Components/CreateNewPoi/validate.js

```diff
@@ -0,0 +1,12 @@
+const validate = (values) => {
```

Comment:
validate ? it doesn't really say what is it doing

Reply:
variable's name changed.